test/Microsoft.Azure.Devices.Edge.Test.Common/Runtime.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Test.Common.Config;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    using Registries = System.Collections.Generic.IEnumerable<(string address, string username, string password)>;
+
+    public class Runtime
```

Comment:
what is difference between Runtime and EdgeDaemon? can it be combined?

Reply:
Runtime is a higher-order class that wraps a lot of the boilerplate code I was seeing in the tests. Specifically, deploying Edge Agent and Edge Hub into a config, and waiting for Edge Hub and Edge Agent to get up and running. So "runtime" refers to our system modules--Edge Agent and Edge Hub--rather than the daemon.

See [IoT Edge Runtime](https://docs.microsoft.com/en-us/azure/iot-edge/iot-edge-runtime) in the docs; sometimes we say "runtime" to refer to the system modules _and_ the daemon, but often it just refers to the two system modules.